packages/db/drizzle/0058_add_mobile_bookmark_click_default_view_mode.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `userSettings` ADD `mobileBookmarkClickDefaultViewMode` text DEFAULT 'reader' NOT NULL;
```

Comment:
Instead of storing this in the database, have you considered storing it with the other mobile local settings in `/apps/mobile/lib/settings.ts`, I feel like it's not one of the those settings that require synchronization as people don't really switch mobile apps that frequently, what do you think?

Reply:
We might not want it now, but I was thinking "might as well".

I'll change the implementation to use local settings to move the PR forward, but feel free to let me know if you change your mind lol.